Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/AbstractSqlEamDb.java

```diff
@@ -176,8 +176,14 @@ public void updateDbInfo(String name, String value) throws EamDbException {
      * @returns New Case class with populated database ID
      */
     @Override
-    public CorrelationCase newCase(CorrelationCase eamCase) throws EamDbException {
+    public synchronized CorrelationCase newCase(CorrelationCase eamCase) throws EamDbException {
         Connection conn = connect();
+        
+        // check if there is already an existing CorrelationCase for this Case
+        CorrelationCase cRCase = getCaseByUUID(eamCase.getCaseUUID());
+        if (cRCase != null) {
+            return cRCase;
+        }
```

Comment:
This should be moved inside the try block, or the Connection needs to be otherwise closed.

Reply:
@rcordovano Unless we are talking about different things, I believe the connection for this call gets opened and closed inside the getCaseByUUID() method.

Reply:
@eugene7646, look at the finally block of the code before the modification. The local conn is closed.

Reply:
@rcordovano oh that connection! Good catch. Done.